clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -287,9 +334,12 @@ func (m *apiCallMetrics) buildKey(resource, subresource, verb, scope string) str
 	return fmt.Sprintf("%s|%s|%s|%s", resource, subresource, verb, scope)
 }
 
-func (ap *apiCallMetric) Validate() error {
+func (ap *apiCallMetric) Validate(allowedSlowCalls int) error {
 	threshold := ap.getSLOThreshold()
 	if err := ap.Latency.VerifyThreshold(threshold); err != nil {
+		if allowedSlowCalls > 0 && ap.SlowCount <= allowedSlowCalls {
```

Comment:
nit: You don't need to check "allowedSlowCalls > 0" - if it's <=0, then the second part will fail anyway.

Reply:
Zero indicates count of slow API calls is turned off and SlowCount won't be populated (and the second check will pass, not fail).

Reply:
Thanks for pointing. I added a comment at that place now, that it would be useful to gather it no matter if the param is set or not - that is interesting metric by itself. When done, this will no longer be needed.

Reply:
Long term this check will be removed and failed count will be measured no matter whether we will use it to make decision on failures or not. 

I've added this guard for now, so we can gradually enable this feature in our tests and minimize breaking things (as per https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/experiments.md)

Reply:
OK - Then please add a TODO in line 191 to get rid of the if and do that unconditionally once we prove it's stable.